html-forms/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" id="email">
+      <div>
+        <label for="coding-cool">Why is coding cool?</label>
+      </div>
+      <select name="options">
```

Comment:
make sure the select has an id attribute which matches the for attribute of the label